Source/Fuse.Scripting.JavaScript/ModuleInstance.uno

```diff
@@ -103,8 +109,6 @@ namespace Fuse.Scripting.JavaScript
 					// Don't report chain-errors of already reported errors
 					if (!se.Message.Contains(ScriptModule.ModuleContainsAnErrorMessage))
 						_diagnostic.SetDiagnostic(se);
-
-					newModuleResult.Dispose();
```

Comment:
Please don't! This was a recent resource-leak fix, and just removing it seems wrong.